### ISSUE TITLE
GO:0000976 transcription regulatory region sequence-specific DNA binding, align with SO

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20791]
'GO:0000976 transcription regulatory region sequence-specific DNA binding' should be reformulated as 
'transcription cis-regulatory region binding' 
(The other rationale is that GO:0000987 cis-regulatory region sequence-specific DNA binding is essentially identical to GO:0000976 as things currently stand)

This will allow including chromatin binding, etc as children of GO:0000976. 

@colinlog 
Thanks, Pascale

**Comment by pgaudet** [comment]
GO:0001067 regulatory region nucleic acid binding
 ->  rename 'transcription regulatory region nucleic acid binding'

**Comment by pgaudet** [comment]
Also - what is the difference between 
- **GO:0000976 transcription regulatory region sequence-specific DNA binding:** Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls transcription of that section of the DNA. The transcribed region might be described as a gene, cistron, or operon.
- and
- **GO:0000987 cis-regulatory region sequence-specific DNA binding:** Interacting selectively and non-covalently with a specific upstream regulatory DNA sequence (transcription factor recognition sequence or binding site) located in cis relative to the transcription start site (i.e., on the same strand of DNA) of a gene transcribed by some RNA polymerase. The proximal promoter is in cis with and relatively close to the core promoter. 
- We dont have any 'trans' regulatory region binding; does that really exist ? ie trans regions exist, but 'trans region binding' implies that a protein links two trans strands; is this really a function?


All children terms should be part of 'cis-regulatory region' DNA binding: 


GO:0000984   bacterial-type RNA polymerase transcription regulatory region   sequence-specific DNA binding
GO:0000987 cis-regulatory region   sequence-specific DNA binding
GO:0001046 core promoter   sequence-specific DNA binding
GO:0001161 intronic transcription   regulatory region sequence-specific DNA binding
GO:0001018 mitochondrial promoter   sequence-specific DNA binding
GO:0001019 plastid promoter transcription   regulatory region sequence-specific DNA binding
GO:0032422 purine-rich negative   regulatory element binding
GO:0001163 RNA polymerase I transcription   regulatory region sequence-specific DNA binding
GO:0000977 RNA polymerase II   transcription regulatory region sequence-specific DNA binding
GO:0001016 RNA polymerase III   transcription regulatory region sequence-specific DNA binding
GO:0001147 transcription termination site   sequence-specific DNA binding

**Comment by pgaudet** [comment]
Or maybe intronic regions are not considered cis ?

**Comment by colinlog** [comment]
the only reason I can think of to keep both 'GO:0000976 transcription regulatory region sequence-specific DNA binding' and 'GO:0000987 cis-regulatory region sequence-specific DNA binding' is the existence of a phenomenon called transvection, whereby a regulatory region one homolog activates the allele on the other homolog. This is pertinent to organisms that pair the paternal and maternal homologs in the cellular nucleosplasm, such as dipteran insects (!). Transvection may account for quite a lot of compound phenotypes in this animal order. In humans, there have been recurrent reports of 'hubs' that may combine multiple regulatory regions. Whether these include inter-chromosomal contacts remains to be established, I think.
In summary, GO:0000987 requires that the interaction between enhancer/promoters to be on the same chromsome, this is often limited to one topologically associated domain (TAD), or two if the gene is at the boundary of two TADs. GO:0000976 does not stipulate _***cis-***. Perhaps GO:0000976 should be a "do not annotate term"? Then (Drosophila) cases of transvection should be housed under a 'transcription activation via regulatory DNA _trans_vection' child term of GO:0000976 while all the other annotations of GO:0000976 could migrate to its child GO:0000987 '_cis-_'?

**Comment by pgaudet** [comment]
but this is a process, not a binding ? Or are there proteins that bind both a cis- and a trans- region?

**Comment by pgaudet** [comment]
Recent paper on transvection: PMID:33479152
But we agree with @colinlog that transvection is a process and that binding is always in cis relative to a promoter.
